doc/reliability.rst

```diff
@@ -0,0 +1,40 @@
+``mybinder.org`` status and reliability
+=======================================
+
+This page serves as a source of information for general Binder status
```

Comment:
Should we try and encourage people to think of mybinder.org as something different from the Binder project and binderhub the software project? I'd change "general Binder status" to "mybinder.org status"